php/utils.php

```diff
@@ -215,7 +215,7 @@ function handle_exception($exception) {
  * @return [boolean] $is_hermes True if Terminus is operating on Hermes
  */
 function is_hermes() {
-  $is_hermes = (TERMINUS_HOST == 'dashboard.getpantheon.com');
```

Comment:
You're right. I don't see any other references. @TeslaDethray: You know of any reason to keep this? Otherwise I'll nuke it.